localstack-core/localstack/utils/aws/client_types.py

```diff
@@ -264,6 +264,7 @@ class ServicePrincipal(str):
     """
 
     apigateway = "apigateway"
+    cloudformation = "cloudformation"
```

Comment:
This is to be used with cloudcontrol. https://docs.aws.amazon.com/cloudcontrolapi/latest/userguide/resource-operations.html#resource-operations-permissions